examples/krypta.rs

```diff
@@ -52,7 +52,6 @@ fn main()
                 }),
             BevyMagicLight2DPlugin,
             ResourceInspectorPlugin::<BevyMagicLight2DSettings>::new(),
-            WorldInspectorPlugin::new(),
```

Comment:
Removed WorldInspectorPlugin for krypta again (I added it in the previous commit) because none of the examples have that one enabled by default